fix: update identities to new schema guide

vinckr opened the pull request:
closes https://github.com/ory/docs/issues/964

Adds guide how to migrate existing identities to a new identity schema


<!--
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.

This text will be included in the changelog. If applicable, include links to documentation or pieces of code.
If your change includes breaking changes please add a code block documenting the breaking change:

```
BREAKING CHANGES: This patch changes the behavior of configuration item `foo` to do bar. To keep the existing
behavior please do baz.
```
-->

## Related Issue or Design Document

<!--
If this pull request

1. is a fix for a known bug, link the issue where the bug was reported in the format of `#1234`;
2. is a fix for a previously unknown bug, explain the bug and how to reproduce it in this pull request;
3. implements a new feature, link the issue containing the design document in the format of `#1234`;
4. improves the documentation, no issue reference is required.

Pull requests introducing new features, which do not have a design document linked are more likely to be rejected and take on average 2-8 weeks longer to
get merged.

You can discuss changes with maintainers either in the Github Discussions in this repository or
join the [Ory Chat](https://www.ory.sh/chat).
-->

## Checklist

<!--
Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of
them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
-->

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [x] I have referenced an issue containing the design document if my change introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added the necessary documentation within the code base (if appropriate).

## Further comments

<!--
If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution
you did and what alternatives you considered, etc...
-->

aeneasr commented on docs/identities/model/manage-identity-schema.mdx:
````diff
@@ -38,15 +40,167 @@ ory patch identity-config {your-project-id} \
+1. Update the identity using the following command:
+
+   ```bash
+   curl -d "@identity-$IDENTITY_ID.json" -X PUT https://$ORY_SLUG.projects.oryapis.com/admin/identities/$IDENTITY_ID \
````
With patch you could quickly patch identities, because you only need to change the schema_id field!

https://www.ory.sh/docs/reference/api#tag/identity/operation/patchIdentity

That way, you don't need the interim step of saving the identity to disk.

aeneasr reviewed (dismissed):
Very nice! A few things I think can be improved

vinckr commented:
made a lot of changes, ready for another look 🙏  @aeneasr 
- added a step by step description how do to it with patch api curl
- changed the go code to use patch and hardcoded the old schema id
- put everything in tabs
- a number of changes to the text
